### ISSUE TITLE
dracut doesn't add serial console driver to initramfs automatically

### ISSUE THREAD

**Issue by rathann** [dracutdevs/dracut#248]
Originally reported as [Fedora bug 1464741](https://bugzilla.redhat.com/show_bug.cgi?id=1464741).

**Description of problem:**
dracut doesn't add serial console driver to initramfs if one is required to drive a console specified in kernel command line. This is related to [Fedora bug 1464709](https://bugzilla.redhat.com/show_bug.cgi?id=1464709). Between kernel 4.10.17 and 4.11.x on F25, the `8250_pci` driver required to drive my serial console was switched from built-in to module in the kernel config, effectively disabling the serial console, because the driver is not loaded before rootfs is mounted and, for example, LUKS password prompt is displayed only on `tty0`. I worked around this by re-running dracut with `--force-drivers 8250_pci`, but this should be done automatically by dracut.

**Version-Release number of selected component (if applicable):**
`dracut-044-78.fc25.x86_64`

**How reproducible:**
Always.

**Steps to Reproduce:**

1. Install any serial port card supported by the `8250_pci` driver, e.g. http://www.i-tec.cz/index.php?t=3&v=103 which is identified as `MosChip Semiconductor Technology Ltd. PCIe 9912 Multi-I/O Controller`
2. Edit the following lines in `/etc/default/grub`:
```
GRUB_TERMINAL_INPUT="console serial"
GRUB_TERMINAL_OUTPUT="console serial"
GRUB_SERIAL_COMMAND="serial --speed=115200 --unit=4 --word=8 --parity=no --stop=1"
GRUB_CMDLINE_LINUX_DEFAULT="LANG=en_GB.UTF=8 console=tty0 console=ttyS4,115200n8"
```
3. Install any 4.11.x (x<7) F25 kernel.
4. Boot the newly installed kernel

**Actual results:**
`8250_pci` driver is not added to initramfs and serial console is not activated after reboot.

**Expected results:**
`8250_pci` driver required for serial console should be added to initramfs if it's specified on the command line so that kernel messages and LUKS password prompt are visible on the console after reboot.

**Additional info:**
This probably affects all Fedora releases with kernel 4.11.x (x<7) where the `8250_pci` driver is built as a module, but I tested only F25.

**Comment by haraldh** [comment]
Thanks!
Fixed with commit c1699b59c6ec0c86f1355fbb68eaf65281b81789

**Comment by rathann** [comment]
That was quick, thank you! Are you planning to release an updated dracut in Fedora soon (for F25)? If yes, then I'll wait for an updated package to test the fix.
